deploy/sre-prometheus/100-ocm-agent-operator.PrometheusRule.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-ocm-agent-operator-alerts
+    role: alert-rules
+  name: sre-ocm-agent-operator-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-ocm-agent-operator-alerts
+    rules:
+    - alert: OCMAgentResponseFailureServiceLogsSRE
+      # Alert if the OCM agent response failure metric has been set for a ten-minute average window
+      expr: avg_over_time(ocm_agent_response_failure{service="service_logs"}[60m]) == 1
+      for: 60m
+      labels:
+        severity: warning
```

Comment:
Since the issue is expected to hit entire fleet, we could consider it `critical` severity? Or the intent is to rollout warning severity first as trial and then promote to critical severity later on?

Reply:
Also, if it's `warning` alert it won't reach SRE at all right?

Reply:
Oh ok :+1: Understood